api/src/Page/Download.php

```diff
@@ -317,6 +319,49 @@ function _csv_report()
         }
     }
 
+    # ------------------------------------------------------------------------
+    # CSV Report of dispensing positions for a plate
+    function _dispensing_csv()
+    {
+        if (!$this->has_arg('cid'))
+            $this->_error('No container id specified');
+        $rows = $this->db->pq("SELECT c.code, s.location, ct.name, ct.capacity, ct.wellperrow,
+                bsp.posx, bsp.posy, si.imagefullpath, si.micronsperpixelx, si.micronsperpixely
+                FROM blsample s
+                INNER JOIN container c ON c.containerid = s.containerid
+                INNER JOIN blsampleimage si ON si.blsampleid = s.blsampleid
+                LEFT OUTER JOIN containertype ct ON (c.containertypeid IS NOT NULL AND c.containertypeid = ct.containertypeid) OR (c.containertypeid IS NULL AND c.containertype = ct.name)
+                LEFT OUTER JOIN
+                    (SELECT * FROM blsampleposition bsp1 WHERE bsp1.blsamplepositionid =
+                        (SELECT MAX(blsamplepositionid) FROM blsampleposition bsp2 WHERE bsp2.blsampleid = bsp1.blsampleid AND bsp2.positiontype='dispensing')
+                    ) bsp ON bsp.blsampleid = s.blsampleid
+                WHERE c.containerid=:1
+                ORDER BY s.location+0",
+                array($this->arg('cid')));
+        $plate = $rows[0];
+        $rowNames = range("A", "H");
+        $dropNames = range("a", "z");
+        // SWISSCI 3 Drop have drops a/c/d
+        if ($plate["NAME"] == "SWISSCI 3 Drop") {
+            $dropNames = array_merge(array("a"), range("c","z"));
+        }
+        $dropsPerWell = $plate["CAPACITY"] / (count($rowNames) * $plate["WELLPERROW"]);
+        $this->app->response->headers->set("Content-type", "application/vnd.ms-excel");
```

Comment:
```suggestion
        $this->app->response->headers->set("Content-type", "text/csv");
```

For compatibility purposes, the accepted standard content type for CSV files is `text/csv`